### ISSUE TITLE
collection paging (page size) not working

### ISSUE THREAD

**Issue by michael-conway** [irods-contrib/metalnx-web#52]
page size setting seems to not work on collection listing

**Comment by michael-conway** [comment]
it's pretty bizarre, the datatables statement sets up paging, and the paging chrome is displayed, but the paging doesn't work.  

The collections view needs a big facelift anyhow, so that it can support actual iRODS paging and stored queries, so in the interest of time I'm proposing to temporarily sidestep this one and take a fresh look at a faster and cleaner display of collections in the next iteration....there are currently a chain of 3 ajax calls to get a listing that is causing some performance issues anyhow...

**Comment by michael-conway** [comment]
iow turn datatables paging off for the 2.0.0 release. the net impact on the users is actually small, and getting datatables paging out of the way will eliminate some level of confusion when the iRODS collections get really big....

**Comment by trel** [comment]
sounds okay to me - just don't want to lose track of it.   bumping to 2.1

**Comment by michael-conway** [comment]
Yeah I committed a patch.  Remember that data table paging is within the
currently retrieved set, not within the iRODS collection, so it's paging a
partial 'page' from iRODS...errrgh.

Worth talking about, but several complaints on rendering speed on
collections browsing so something with a focus on performance and 'true'
iRODS paging needed.

**Comment by sgworth** [comment]
I am OK with this, but doing this cuts off the ability to sort on columns.
That was why we put datatables in.  Sorting was a heavily requested
feature.  And for the record when people start working with collections
containing 15K+ files you will find we start getting server 500 errors
because the browser cache overloads.  Another reason why you see pagination
and not infinity scrolling.

When we look at this for R2.1 we need to make sure that the solution do
allow column sorting and does not break with massive collections.  And
before you give me grief about the silliness of having 15K files in a
collection this requirement came up because when we interviewed LSHC people
they told us it was not uncommon for a data "dumping" area to have over 35K
objects before the process of categorizing started.

**Comment by michael-conway** [comment]
it should allow column sorting, the datatables paging is just set to false,
everything else is the same

**Comment by michael-conway** [comment]
push out to coll browser redesign, this can start as soon as we get search in as we'll be able to use proper ajax and vue.js to accomplish this, folding in functionality we had in cloud browser

**Comment by trel** [comment]
let's leave everything in a milestone, and then bump milestones as necessary.

**Comment by trel** [comment]
Closing.   Much time has passed.   We'll address in any refactoring.
